utils/gyb.py

```diff
@@ -1079,6 +1079,20 @@ def execute_template(
 
 
 def main():
+    """
+    Lint this file.
+    >>> import subprocess, sys
+    >>> script_path = os.path.realpath(__file__)
+    >>> utils_directory = os.path.dirname(script_path)
+    >>> linting = subprocess.Popen([sys.executable,
+    ...                             os.path.join(utils_directory,
+    ...                                         'python-lint'),
+    ...                             script_path])
```

Comment:
Sure. I held off doing this, as we couldn't directly import `python-lint` as it has a `-` in it's name, and **also** doesn't have the `.py` suffix.

However, I've adopted a much cleaner python approach